pbrd/pbr_vty.c

```diff
@@ -29,89 +29,60 @@
 #include "pbrd/pbr_vty_clippy.c"
 
 /* clang-format off */
-DEFPY(pbr_map_match_pcp, pbr_map_match_pcp_cmd, "[no] match pcp <(0-7)$pcp>",
-      NO_STR
-      "Match spec follows\n"
-      "Match based on 802.1p Priority Code Point (PCP) value\n"
-      "PCP value to match\n")
+DEFPY  (pbr_set_table_range,
+	pbr_set_table_range_cmd,
+	"pbr table range (10000-4294966272)$lb (10000-4294966272)$ub",
+	PBR_STR
+	"Set table ID range\n"
```

Comment:
I think this should be renamed because it we have more commands under `table`, not only `range`, then it's misleading and most likely gonna be missed to change.

Reply:
"pbr table range" is an existing command since at least five years ago (this change just moved some code around in the file). Should your proposed syntax change be under a different PR?